### DISCUSSION TITLE
Add switch for climate control

### DISCUSSION THREAD

**Discussion by mouth4war** [jseidl/magic-areas#197] [Ideas / Feature Requests]
Similar to light control switch

**Comment by jseidl** [comment]
Sounds reasonable! Will look into after `3.x` release (pretty soon[tm])

**Reply by mouth4war** [reply]
Pretty please 😁

**Comment by jseidl** [comment]
Good news folks! https://github.com/jseidl/hass-magic_areas/pull/344

Coming up next release

**Comment by jseidl** [comment]
This is implemented in release 4.0.0 which is out!
